proto/ethermint/evm/v1/evm.proto

```diff
@@ -30,6 +30,9 @@ message Params {
   // active_static_precompiles defines the slice of hex addresses of the precompiled
   // contracts that are active
   repeated string active_static_precompiles = 10;
+  // denom_decimals defines the amount of decimals of the evm denom
+  uint32 denom_decimals = 11
```

Comment:
question: will ATOM be considered the evm denom when migrating to ICS?

Reply:
Yes in the upgrade handler for v20 we will change the EVM denom to the IBC voucher of Atom.